### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Frosendroska**

### CODE REVIEW

**Review by AnastasiiaGorelova — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,6 +10,11 @@
 4. Run `gw dockerComposeUp` to enable proxy
 5. Open `nnbuilder.ipr` file in Intellij (on macOS run `open *.ipr`)
 
+## Install database on MAC
+ * brew install postgres
+ * pg_ctl -D /usr/local/var/postgres start
+ * psql postgres
```
Comment (iskhakovt) [inline README.md]: Why do you need to run the cli?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# nnbuilder ![Build Status](https://img.shields.io/circleci/build/github/hse-mentorship/nnbuilder?style=flat-square) ![Lines of Code](https://img.shields.io/tokei/lines/github/hse-mentorship/nnbuilder?style=flat-square)
+# nnbuilder ![Build Status](https://img.shields.io/circleci/build/github/hse-mentorship/nnbuilder?style=flat-square)
```
Comment (iskhakovt) [inline README.md]: Why do you want the line count to land on a separate line?

Comment (Frosendroska) [reply README.md]: just because

File: README.md (hunk 1 above)
Comment (Frosendroska) [reply README.md]: this is just a tutorial to make a DB in localhost...

**Pull request merged**